modules/create_timepoint/help/create_timepoint.md

```diff
@@ -1,5 +1,5 @@
 # Create Timepoint
 
-To add a new time-point, enter the subproject (or cohort) for this visit, followed by "Site" and "Visit Label". Once the required fields are complete, click the “Create Time Point” button to register this time-point. Users should see a success message, and then click on “Click here to continue”.
+To add a new timepoint, select the *Subproject* for this visit, which will reveal new fields. Select the appropriate *Site*, *Project*, and *Visit Label*. Then, click **Create Timepoint** to register this timepoint. You should see a success message, including a clickable link that says “Click here to continue.”
 
-Note: after creating a new time-point, users will wish to returned to the Time-Point list, and click on the new visit from the List of Visits table.  Inside the new Timepoint page, click on “Start Next Stage” in the left sidebar to begin the step that will add instruments to this timepoint based on a defined battery.
\ No newline at end of file
+Note: after creating a new timepoint, you will return to the _Timepoint List_ page. Click on the new visit in the table, and then in the next page, click on **Start Visit Stage** in the left sidebar to specify the visit date, which will automatically  add instruments to this timepoint based on the appropriate behavioural battery of tests.
```

Comment:
I think the part about needing to click "Start Next Stage" in order for the battery to be populated is important or users are likely to get confused.

Reply:
do you think it needs to be worded formatted differently so it stands out more? what do you suggest?

Reply:
The note was just showing up as deleted in the diff when I reviewed it, it's fine now that it's there.